sapl/templates/sessao/adicionar_varias_materias_expediente.html

```diff
@@ -126,5 +143,22 @@ <h3>{% trans 'Pesquisa concluída com sucesso! Foi encontrada 1 matéria.'%}</h3
 		$(window).on('beforeunload', function() {
       $("input[type=submit], input[type=button]").prop("disabled", "disabled")
     });
+    function checkAll(elem) {
+      let checkboxes = document.getElementsByName('materia_id');
+      for (let i = 0; i < checkboxes.length; i++) {
+        if (checkboxes[i].type == 'checkbox')
+            checkboxes[i].checked = elem.checked;
+      }
```

Comment:
Usar jQuery ao invés de JavaScript puro. O código fica mais conciso e legível.

```suggestion
$('input[name="materia_id"]:checkbox').prop('checked', elem.checked);
```